package.json

```diff
@@ -40,5 +42,8 @@
     "docs",
     "examples/*",
     "packages/*"
-  ]
-}
\ No newline at end of file
+  ],
+  "dependencies": {
+    "react-hook-form": "^7.52.1"
```

Comment:
I think all dependencies you've added should be in packages/identitykit/package.json.
Not in the root.